common/src/main/java/io/druid/metadata/MetadataStorageConnector.java

```diff
@@ -46,4 +46,6 @@ public Void insertOrUpdate(
   public void createTaskTables();
 
   public void createAuditTable();
+
+  public void createNamespacesTable();
```

Comment:
I'm considering moving this to a module, so don't get caught up on it being in mainline druid